src/components/BlogDetails/BlogDetailsComponents/BlogComment.jsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import Image from "next/image";
+import { comments } from "@/data/blogsData";
+
+const CommentSection = () => {
+  return (
+    <div className="max-w-5xl mx-auto bg-white shadow-md rounded-lg p-6">
+      <h2 className="text-2xl font-semibold mb-6">
+        Total Comment ({comments.length.toString().padStart(2, "0")})
```

Comment:
Please use like this **convertNumToPad(size(comments) || 0)**  helper function from appHelper.

Please do import size from lodash

Reply:
i tried this way but it's not working **Total Comment ({convertNumToPad(comments.length)})**